modules/S22_php_check.sh

```diff
@@ -74,7 +80,77 @@ s22_script_check() {
     else
       COMMON_FILES_FOUND=""
     fi
-    print_output "[+] Found ""$ORANGE""parsing issues""$GREEN"" in script ""$COMMON_FILES_FOUND"":""$NC"" ""$(print_path "$LINE")" "" "$PHP_LOG"
+    print_output "[+] Found ""$ORANGE""parsing issues""$GREEN"" in script ""$COMMON_FILES_FOUND"":""$NC"" ""$(print_path "$LINE")"
     echo "$VULNS" >> "$TMP_DIR"/S22_VULNS.tmp
   fi
 }
+
+add_recommendations(){
+   local VALUE
+   local KEY
+   VALUE="$1"
+   KEY="$2"
+
+   if [[ $VALUE == *"M"* ]]; then
+      LIMIT="${VALUE//M/}"
+   fi
+
+   if [[ $KEY == *"memory_limit"* ]] && [[ $(( LIMIT)) -gt 50 ]]; then
+     return 1
+   elif [[ $KEY == *"post_max_size"* ]] && [[ $(( LIMIT)) -gt 20 ]]; then
+     return 1
+   elif [[ $KEY == *"max_execution_time"* ]] && [[ $(( LIMIT )) -gt 60 ]]; then
+     return 1
+   else
+     return 0
+   fi
+}
+
+s22_check_php_ini(){
+  local PHP_INI_FAILURE
+  local PHP_INI_LIMIT_EXCEEDED
+  local PHP_INI_WARNINGS
+  PHP_INI_FAILURE=0
+  PHP_INI_LIMIT_EXCEEDED=0
+  PHP_INI_WARNINGS=0
+
+  mapfile -t PHP_INI_FILES < <( find "$FIRMWARE_PATH" -xdev "${EXCL_FIND[@]}" -iname 'php.ini' -exec md5sum {} \; 2>/dev/null | sort -u -k1,1 | cut -d\  -f3 )
+  for PHP_FILE in "${PHP_INI_FILES[@]}" ;  do
+    print_output "[*] iniscan check of ""$(print_path "$PHP_FILE")"
+    mapfile -t INISCAN_RESULT < <( "$PHP_SCAN_PATH" scan --path="$PHP_FILE" )
+    for LINE in "${INISCAN_RESULT[@]}" ; do  
```

Comment:
could you also remove some other iniscan output:
On command line I would filter with grep the following way: 
| grep -v "Results for" | grep -v "passing" | grep -v "failure" | grep -v "======" | grep -v "PHP Warning" | grep -v "Executing INI Scan"
You can probably also do it in your parsing routine from INISCAN_RESULT array